From f93975f4d0c9161926953483d676f0a0f78c2620 Mon Sep 17 00:00:00 2001
From: Dmitry Shachnev <mitya57@gmail.com>
Date: Tue, 18 Apr 2023 13:09:27 +0300
Subject: [PATCH 081/127] Fix capitalization error in auto-generated
 qdbusmacros.h include

https://bugs.launchpad.net/bugs/2016703

Pick-to: 6.5 6.2 5.15
Change-Id: I521c7b66d6c1c27ea790f0564b175cccb0027802
Reviewed-by: Alexandru Croitor <alexandru.croitor@qt.io>
(cherry picked from commit dca0304c26012a57abae79c9b82f6cbdd0467018)
---
 sync.profile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sync.profile b/sync.profile
index 0292bf0dc2..7dd0177b90 100644
--- a/sync.profile
+++ b/sync.profile
@@ -77,7 +77,7 @@
         "qsql.h" => "QtSql/qtsqlglobal.h"
     },
     "QtDBus" => {
-        "qdbusmacros.h" => "QtDbus/qtdbusglobal.h"
+        "qdbusmacros.h" => "QtDBus/qtdbusglobal.h"
     },
     "QtTest" => {
         "qtest_global.h" => "QtTest/qttestglobal.h"
-- 
2.46.0

